appveyor.yml

```diff
@@ -42,8 +42,8 @@ install:
     - "python --version"
     - "cython --version"
 
-    # Install pytest-fits
-    - "pip install https://github.com/astrofrog/pytest-fits/archive/master.zip"
+    # Install pytest-arraydiff
+    - "pip install https://github.com/astrofrog/pytest-arraydiff/archive/master.zip"
```

Comment:
if you like, this can be added to PIP_DEPENDENCIES

Reply:
I'll do it in a separate PR, I've been meaning to do a pytest-arraydiff release so will update that at the same time